Source/Core/UICommon/CMakeLists.txt

```diff
@@ -5,15 +5,19 @@ set(SRCS
   USBUtils.cpp
 )
 
-if(USE_X11)
-  set(SRCS ${SRCS} X11Utils.cpp)
-endif()
-
 set(LIBS common cpp-optparse)
 if(LIBUSB_FOUND)
   set(LIBS ${LIBS} ${LIBUSB_LIBRARIES})
 endif()
 
+if(USE_X11)
+  find_package(GTK2 REQUIRED)
```

Comment:
You're right. Moved some things around, these should be forwarded from wxWidgets now.